### PR TITLE
Remove dedup generic v2

### PR DESCRIPTION
**Pull request by tomneeman151293**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/34080

## Description
Fixing test failure.
Reason is playbook V2 is deprecated, so V2 test should be as well
Removed the usages of the V2 test from many places which were unnecessary,
and moved V2 playbook to deprecated content, and V2 test playbook to deprecated test playbooks

## Screenshots
Paste here any images that will help the reviewer

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [x] Documentation

### CODE REVIEW

**Comment by tomneeman151293** [comment]
Validation is failing on updating docker image for demisto/machine-learning
after talking to Roy Sagi he said its better to ignore that error, as there is no docker tags for this kind of images currently.

**Comment by ShahafBenYakir** [comment]
@tomneeman151293 Are we waiting for something else here?>

**Comment by tomneeman151293** [comment]
@ShahafBenYakir You did not approve it.
Also it will be needed to force-merge because validation fails it although it shouldn't

**Review by ShahafBenYakir — COMMENTED** [review]
As customers don't know what are test PBs, in all three RN files (base, common playbooks, common scripts) need to change the text to:
```
- Maintenance and stability enhancements.
```

**Comment by ShahafBenYakir** [comment]
Also need to pull from master

File: Utils/tests/id_set.json [hunk 1]
```diff
@@ -4002,7 +4002,7 @@
                     "getIncidents"
                 ],
                 "tests": [
-                    "Dedup - Generic v2 - Test"
+                    "No test"
```
[the remark this answers is not in the document]
Comment (tomneeman151293) [reply Utils/tests/id_set.json:4005]: so what should i do? leave as is? @ShahafBenYakir

[the remark this answers is not in the document]
Comment (ShahafBenYakir) [reply Utils/tests/id_set.json:4005]: Yes, we don't edit the id_set manually

**Comment by tomneeman151293** [comment]
@ShahafBenYakir  Only failure is on validation due to the machine learning docker image not containing tag, see my comment above. Can merge I guess

**Review by ShahafBenYakir — APPROVED** [review]

**Comment by ShahafBenYakir** [comment]
Force merging due to invalid Docker error

**Pull request merged**